### PR TITLE
STENCIL-3261 - Move Google AMP settings to its own section

### PR DESCRIPTION
**Pull request by bookernath**
#### What?

This should help us avoid confusion as most merchants are not in the Google AMP beta.

#### Tickets / Documentation

Add links to any relevant tickets and documentation.

[STENCIL-3261](https://jira.bigcommerce.com/browse/STENCIL-3261)

#### Screenshots

image

### CODE REVIEW

**Comment by mjschock** [comment]
LGTM; 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
 - Print a readable error instead of dumping the whole error object to the console [#950](https://github.com/bigcommerce/cornerstone/pull/950)
 - Fixed homepage featured products floating left and unecessarily wrapping to next row [#948](https://github.com/bigcommerce/cornerstone/pull/948)
 - Add google recaptcha v2 support to cornerstone. [#951](https://github.com/bigcommerce/cornerstone/pull/951)
+- Add google recaptcha v2 support to cornerstone. [#951](https://github.com/bigcommerce/cornerstone/pull/951)
```
Comment (junedkazi) [inline CHANGELOG.md]: Why is there a duplicate entry here ?

Comment (bookernath) [inline CHANGELOG.md]: Woops, yep. Moments...

Comment (bookernath) [inline CHANGELOG.md]: Fixed!

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@
 - Print a readable error instead of dumping the whole error object to the console [#950](https://github.com/bigcommerce/cornerstone/pull/950)
 - Fixed homepage featured products floating left and unecessarily wrapping to next row [#948](https://github.com/bigcommerce/cornerstone/pull/948)
 - Add google recaptcha v2 support to cornerstone. [#951](https://github.com/bigcommerce/cornerstone/pull/951)
+- Move Google AMP settings to a new section of theme editor. [#956](https://github.com/bigcommerce/cornerstone/pull/956)
```
Comment (junedkazi) [inline CHANGELOG.md]: Not sure we need new change log entry for for changes to a feature which is not yet released. We should just let all of this roll into the parent entry.

**Pull request merged**